### ISSUE TITLE
Output is confused when using multiple browsers

### ISSUE THREAD

**Issue by c24w** [prantlf/karma-brief-reporter#4]
Hey, I've just come across your plugin and I'm a big fan!

I noticed that it goes a bit crazy when multiple browsers are involved, because the browsers are contesting for console output:

![karma-brief-reporter-multiple-browsers](https://user-images.githubusercontent.com/710406/46149461-4a10d200-c262-11e8-95a7-3577650e2d05.gif)

I'm not familiar with karma plugins, but from a quick play around I can see that each `browser` has an `id`, so it might be possible to accumulate each `browser.lastResult` against its `browser.id`, in the `onSpecComplete` method.  The stats per browser could them be split or aggregated to build the console output.

Edit: I've just noticed you're already doing something similar to what I'm suggesting to aggregate browser `console.logs`!

**Comment by prantlf** [comment]
Thanks for reporting this! I do not test with multiple browsers at the same time. I aggregate browser logs 
 by `browser.id`, but the progress output on the console is not aware of multiple browsers. I'm looking into it.

**Comment by prantlf** [comment]
I finally got to this. Karma does not pass the list of all browsers to the initial `onRunStart` event. It causes the initial progress printed for fewer browsers, than configured, but as soon as all browsers are running, the output is consistent:

    Chrome       6 total        5 passed      1 failed      0 skipped
    Firefox      2 pending      3 passed      0 failed      0 skipped

I added support for multiple browsers with eef4cebc651299695d83dc86c0c559862632731d and released in the version [0.2.0](https://github.com/prantlf/karma-brief-reporter/releases/tag/v0.2.0).
